src/pages/HomePage/HomePage.js

```diff
@@ -185,9 +193,9 @@ class HomePage extends Component {
 									margin: 0,
 								}}
 							>
-								All Apps
+								{sortedApps.length ? 'All Apps' : 'Create an app to get Started'}
```

Comment:
I don't think this is the best way to represent "Create a new app to get started". We need something similar to the `NoMatch` page here.